site/app/libraries/database/DatabaseQueries.php

```diff
@@ -3283,6 +3300,29 @@ public function getPeerGradingAssignment($gradeable_id) {
         }
         return $return;
     }
+    
+    /**
+     * Adds an assignment for someone to get all the peer feedback for a given gradeable
+     *
+     * @param string $gradeable_id
+     */
+    public function getPeerFeedback($gradeable_id) {
+        $this->course_db->query("SELECT grader_id, user_id, feedback FROM peer_feedback WHERE g_id = ? ORDER BY grader_id", [$gradeable_id]);
```

Comment:
Same as above, you seem to be `user_id` here without considering `team_id`. But now that I think of it, what does it mean for a team to give a grader feedback? Is that actually how we want this to work?

Reply:
As discussed, we will use the user_id of the student giving feedback